sway-lib-std/src/array_conversions/b256.sw

```diff
@@ -66,10 +70,30 @@ impl b256 {
     ///     assert(x == 0x0102030405060708090a0b0c0d0e0f101112131415161718191a1b1c1d1e1f20);
     /// ```
     pub fn from_le_bytes(bytes: [u8; 32]) -> Self {
-        let a = u64::from_le_bytes([bytes[0], bytes[1], bytes[2], bytes[3], bytes[4], bytes[5], bytes[6], bytes[7]]);
-        let b = u64::from_le_bytes([bytes[8], bytes[9], bytes[10], bytes[11], bytes[12], bytes[13], bytes[14], bytes[15]]);
-        let c = u64::from_le_bytes([bytes[16], bytes[17], bytes[18], bytes[19], bytes[20], bytes[21], bytes[22], bytes[23]]);
-        let d = u64::from_le_bytes([bytes[24], bytes[25], bytes[26], bytes[27], bytes[28], bytes[29], bytes[30], bytes[31]]);
+        let a = u64::from_le_bytes(
+            [
+                bytes[0], bytes[1], bytes[2], bytes[3], bytes[4],
+                bytes[5], bytes[6], bytes[7],
+            ],
+        );
+        let b = u64::from_le_bytes(
+            [
+                bytes[8], bytes[9], bytes[10], bytes[11], bytes[12],
+                bytes[13], bytes[14], bytes[15],
+            ],
+        );
+        let c = u64::from_le_bytes(
+            [
+                bytes[16], bytes[17], bytes[18], bytes[19], bytes[20],
+                bytes[21], bytes[22], bytes[23],
+            ],
+        );
+        let d = u64::from_le_bytes(
+            [
+                bytes[24], bytes[25], bytes[26], bytes[27], bytes[28],
+                bytes[29], bytes[30], bytes[31],
+            ],
+        );
```

Comment:
This looks a little too long vertically:

<img width="569" alt="image" src="https://github.com/FuelLabs/sway/assets/20915464/f511a87b-ae3b-4d78-b369-d20809765913">

This is the same piece code under `rustfmt`, might not be worth to block but we can discuss this one.

<img width="803" alt="image" src="https://github.com/FuelLabs/sway/assets/20915464/780bcf0e-87bf-4494-a513-542f57a26587">

Reply:
Great!